modules/client/src/controllers/BuyController.ts

```diff
@@ -69,6 +69,7 @@ export default class BuyController extends AbstractController {
           // PUNT on this -- AB
           break
         case 'PT_CHANNEL':
+        case 'PT_CUSTODIAL':
```

Comment:
So my guess is this won't cause any material problems, but it should be noted this type isn't implemented hub-side with this PR

Reply:
Yep, should just fail on the hub immediately.